File: Problems/1502. (E) Can Make Arithmetic Progression From Sequence/Python3/solution.py
class Solution:
  def canMakeArithmeticProgression(self, arr: List[int]) -> bool:
    # sort

    arr.sort()
    for i in range(2, len(arr)):
      if arr[i] - arr[i-1] != arr[i-1] - arr[i-2]:
        return False
        
    return True

File: Problems/1502. (E) Can Make Arithmetic Progression From Sequence/Java/solution.java
class Solution {
  public boolean canMakeArithmeticProgression(int[] arr) {
    // sort

    Arrays.sort(arr);
    for (int i = 2; i < arr.length; i++) {
      if (arr[i] - arr[i-1] != arr[i-1] - arr[i-2])
        return false;
    }
    return true;
  }
}
